unity/ARUI/Assets/MixedRealityToolkit.Generated/ProjectPreferences.asset

```diff
@@ -24,7 +24,7 @@ MonoBehaviour:
     keys:
     - _MixedRealityToolkit_Editor_ConfiguratorState
     - MixedRealityToolkit_Editor_AudioSpatializerCount
-    values: f401000002000000
+    values: c800000002000000
```

Comment:
What preference change does this reflect? If unknown, I'm not sure we should include the change.

Reply:
Ah, that shouldn't have been added. Will resolve.